### ISSUE TITLE
Allow Github markdown, mainly enter = new line

### ISSUE THREAD

**Issue by Terrox** [icewind1991/files_markdown#69]
Would be good to let the enter key = new line. Frustrating to have pasted text become mangled by unwrapped lines. Having to remember to double-enter is not fun.

**Comment by icewind1991** [comment]
The "github style" newline handling is something I'm hesitant to add given that it can break the rendering of existing documents

**Comment by Terrox** [comment]
It would have to be a config option for that reason, but I think a lot of people would appreciate it.

**Comment by mr-bolle** [comment]
I compare a lot of different markdown editor with this one.
And found a way how it works, you *end the line* with **two or more spaces** and type return.

Could you implement this way to create a new line `<br />` or `<br>`
[Syntax daringfireball.net](https://daringfireball.net/projects/markdown/syntax#p)

**Comment by Terrox** [comment]
Double space is not as good as just pressing enter, no training required.

**Comment by mr-bolle** [comment]
You are right, but the syntax of Markdown should be adjusted, more information in the link above.

**Comment by Isvvc** [comment]
> no training required.

Markdown itself requires a small amount of training, and removing that would make it less efficient. This app is following the Markdown standards as it ought to.

**Comment by Terrox** [comment]
Disagree for users simply reporting bugs in plain text. They need to be able to separate lines of text and cannot do this currently without remembering to double-enter and they never remember. They can even paste screenshots without any training, copy paste just works.

**Comment by siriak** [comment]
Any updates on that topic?
